src/app/dashboard/offerings/components/OfferingItem.jsx

```diff
@@ -0,0 +1,78 @@
+import { Accordion, AccordionContent, AccordionTitle, AccordionPanel, Badge } from 'flowbite-react'
+import { HiIdentification, HiCalendar, HiLink, HiOutlineScale } from 'react-icons/hi'
+import OfferingActions from './OfferingActions'
+
+export default async function OfferingItem ({ offering, offeringUrl }) {
```

Comment:
Suggestion: following my previous comment
```suggestion
export default async function OfferingItem ({ offering }) {
```